torchvision/models/vgg.py

```diff
@@ -60,31 +66,31 @@ def _initialize_weights(self):
                 nn.init.constant_(m.bias, 0)
 
 
-def make_layers(cfg, batch_norm=False):
+def make_layers(cfg: List[Union[str, int]], batch_norm: bool = False) -> nn.Sequential:
     layers = []
     in_channels = 3
     for v in cfg:
-        if v == 'M':
-            layers += [nn.MaxPool2d(kernel_size=2, stride=2)]
-        else:
-            conv2d = nn.Conv2d(in_channels, v, kernel_size=3, padding=1)
+        if isinstance(v, int):
+            conv2d = nn.Conv2d(in_channels, int(v), kernel_size=3, padding=1)
```

Comment:
Just to verify: `mypy` complained about `v` being either `str` or `int` and thus we changed the conditional, correct? If that is the case isn't the cast to `int` here obsolete?

```suggestion
        if isinstance(v, int):
            conv2d = nn.Conv2d(in_channels, v, kernel_size=3, padding=1)
```

Reply:
Correct, I forgot to remove this! Just fixed it